flang/lib/Lower/OpenMP.cpp

```diff
@@ -125,6 +125,189 @@ createBodyOfOp(Op &op, Fortran::lower::AbstractConverter &converter,
     privatiseVars(converter, *clauses);
 }
 
+static std::int64_t
+createOrderedDependSourceOp(Fortran::lower::AbstractConverter &converter,
+                            Fortran::lower::pft::Evaluation &eval,
+                            SmallVectorImpl<Value> &operands) {
```

Comment:
nit: Most (all?) of the lowering code (unless it is clearly wrapped in a `namespace`, as in a header) explicitly spells out the full namespace path. (Here, it would be `llvm::SmallVectorImpl<mlir::Value>`, I assume.)

Reply:
Done.